fonzie/views/status.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+API status views
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from fonzie import __version__ as fonzie_version
+
+
+class VersionView(APIView):
+    """
+    API endpoint to get the running API version
```

Comment:
so here should you write 
"""API endpoint to get the running API version"""
(ref to our talk this morning :-)

Reply:
I'll fix this in another PR.